app/views/sessions/_filters.html.erb

```diff
@@ -3,28 +3,87 @@
 <div class="flex justify-between w-full mb-6">
   <div class="flex items-center gap-2 overflow-x-scroll no-scrollbar">
     <%= link_to(
-      'All dates',
-      url_for(resource),
-      class: [
-        "flex min-w-fit items-center justify-center px-4 py-2 rounded-full font-medium",
-        "bg-red text-white": params[:starts_at].nil?,
-        "bg-white text-red": params[:starts_at].present?
-      ]
-    ) %>
+          'All dates',
+          url_for(resource),
+          class: [
+            "flex min-w-fit items-center justify-center px-4 py-2 rounded-full font-medium",
+            "bg-red text-white": params[:starts_at].blank?,
+            "bg-white text-red": params[:starts_at].present?
+          ]
+        ) %>
     <% current_conference.session_dates.each do |session_date| %>
       <%= link_to(
-        session_date.strftime("%a, %d"),
-        url_for([resource, starts_at: session_date]),
-        class: [
-          "flex min-w-fit items-center justify-center px-4 py-2 font-medium rounded-full",
-          "bg-red text-white": params[:starts_at] == session_date.to_s,
-          "bg-white text-red": params[:starts_at] != session_date.to_s
-        ]
-      ) %>
+            session_date.strftime("%a, %d"),
+            url_for([resource, starts_at: session_date]),
+            class: [
+              "flex min-w-fit items-center justify-center px-4 py-2 font-medium rounded-full",
+              "bg-red text-white": params[:starts_at] == session_date.to_s,
+              "bg-white text-red": params[:starts_at] != session_date.to_s
+            ]
+          ) %>
     <% end %>
   </div>
 
-  <div class="flex items-center justify-center ml-3 p-2 bg-white rounded-full min-w-[40px] min-h-[40px]">
-    <%= inline_svg_tag("icons/filters.svg", size: "18") %>
+  <div class="flex items-center justify-center ml-3 p-2 rounded-full min-w-[40px] min-h-[40px] <%= active_filters? ? "bg-red text-white" : "bg-white text-red" %>">
+    <div class="relative" data-controller="dropdown">
+      <button type="button" data-action="dropdown#toggle click@window->dropdown#hide" class="flex items-center justify-center p-2 rounded-full">
+        <%= inline_svg_tag("icons/filters.svg", size: "18") %>
+      </button>
+
+      <div
+        data-dropdown-target="menu"
+        class="hidden transition transform origin-top-right absolute right-0"
+        data-transition-enter-from="opacity-0 scale-95"
+        data-transition-enter-to="opacity-100 scale-100"
+        data-transition-leave-from="opacity-100 scale-100"
+        data-transition-leave-to="opacity-0 scale-95">
+        <div class="pt-2 pb-2 bg-white rounded-lg shadow-[0_10px_50px_0_rgba(0,0,0,0.40)] flex-col inline-flex absolute top-4 right-[-8px] right-0 w-40 max-w-40">
+          <%= form_with url: sessions_path, class: "grid", method: :get do |form| %>
+            <%= form.hidden_field :starts_at, value: params[:starts_at] %>
```

Comment:
let's use a hidden_field to maintain the active filters when querying over a different date. Currently, the active filters will be lost if a new date is selected
![image](https://github.com/user-attachments/assets/489e0dd9-d81a-4f1a-acf8-1605fd43a59b)

Reply:
Reminder - this one needs to be addressed